tools/archetype/binding.test/src/main/resources/archetype-resources/META-INF/MANIFEST.MF

```diff
@@ -7,22 +7,18 @@ Bundle-Version: ${version.replaceAll("-SNAPSHOT", ".qualifier")}
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Fragment-Host: ${package}
 Import-Package: ${package},
- ${package}.internal,
+ ${package}.handler,
  org.slf4j,
- groovy.json,
- groovy.lang,
- org.codehaus.groovy.reflection,
- org.codehaus.groovy.runtime,
- org.codehaus.groovy.runtime.callsite,
- org.codehaus.groovy.runtime.typehandling,
  org.eclipse.smarthome.core.events,
  org.eclipse.smarthome.core.thing.util,
- org.eclipse.smarthome.test,
  org.eclipse.smarthome.test.storage,
+ org.eclipse.smarthome.test.java,
  org.hamcrest;core=split,
+ org.mockito,
  org.junit;version="4.0.0",
  org.osgi.service.device,
  org.osgi.framework
+Require-Bundle: org.junit,org.mockito,org.hamcrest
```

Comment:
> Is it only needed for hamcrest, or do the other two need to be included there as well?

From my experience with these libs all three are mandatory to make this work. And from my memory its not only about hamcrest but mockito, too.
The error when removing one or the other from the Required-Bundles is something like:
```
Access restriction: The method 'OngoingStubbing<Bridge>.thenReturn(Bridge)' is not API (restriction on required library '/Users/henning/work/smarthome-master2/ws/.metadata/.plugins/org.eclipse.pde.core/.bundle_pool/plugins/org.mockito_1.9.5.v201311280930.jar')
```